heutagogy/views.py

```diff
@@ -46,6 +45,19 @@ def update_query(url, params):
     return urlparse.urlunparse(url_parts)
 
 
+def filter_url(url):
```

Comment:
I would improve a naming and make the function more generic.

```python
def filter_query_string(url, query_string):
    ...

def filter_utm_query_string(url):
    return filter_query_string(url, 'utm_')
```

Reply:
This function also removes fragments (`#blah` part in URLs), and sorts query parameters. This is more like a canonical representation of URL in the system, so database text search is more predictable.